### PR TITLE
zelenaya.net

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/zelenaya.net.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!--
+efremov.zelenaya.net self signed
+tambov.zelenaya.net self signed
+elets.zelenaya.net self signed
```
Comment (J0WI) [inline src/chrome/content/rules/zelenaya.net.xml]: Please double check for cert chain issues.

Comment (galeksandrp) [inline src/chrome/content/rules/zelenaya.net.xml]: Fixed.

Comment (J0WI) [inline src/chrome/content/rules/zelenaya.net.xml]: Nope, they are still treated as self-signed. They just had chain issues, but now they expired recently. Let's see if they get fixed.

Comment (J0WI) [inline src/chrome/content/rules/zelenaya.net.xml]: @galeksandrp they are still treated as self-signed

**Pull request closed without merging**